RecoParticleFlow/Configuration/python/RecoParticleFlow_EventContent_cff.py

```diff
@@ -143,8 +143,19 @@ def _modifyPFEventContentForHGCalFEVT( obj ):
 
 
 # mods for HGCAL
+
+_phase2_RecoParticleFlowFEVT_outputCommands = RecoParticleFlowFEVT.outputCommands
+_phase2_RecoParticleFlowFEVT_outputCommands.append('keep recoPFRecHits_particleFlowClusterECAL__*')
+_phase2_RecoParticleFlowFEVT_outputCommands.append('keep recoPFRecHits_particleFlowClusterECAL_Cleaned_*')
+
+_phase2_RecoParticleFlowRECO_outputCommands = RecoParticleFlowRECO.outputCommands
+_phase2_RecoParticleFlowRECO_outputCommands.append('keep recoPFRecHits_particleFlowClusterECAL_Cleaned_*')
+
+_phase2_RecoParticleFlowAOD_outputCommands = RecoParticleFlowAOD.outputCommands
+_phase2_RecoParticleFlowAOD_outputCommands.append('keep recoPFRecHits_particleFlowClusterECAL_Cleaned_*')
+
```

Comment:
These also have the problem of not being a copy and instead modifying the original.